src/liveEditor/index.ts

```diff
@@ -225,11 +233,28 @@ export class VisualEditor {
             });
         }
     };
+    handleCursorPosition = (event: MouseEvent): void => {
+        if (this.customCursor) {
+            const mouseY = event.clientY;
+            const mouseX = event.clientX;
+
+            this.customCursor.style.left = `${mouseX}px`;
+            this.customCursor.style.top = `${mouseY}px`;
+        }
+    };
     hideCustomCursor = (): void => {
         if (this.customCursor) {
             this.customCursor.classList.remove("visible");
         }
     };
+    resetCustomCursor = (): void => {
+        if (this.customCursor) {
+            generateCustomCursor({
+                fieldType: "empty",
+                customCursor: this.customCursor,
+            });
+        }
+    };
```

Comment:
Since the custom cursor code is growing, we could consider moving them to a separate file.